apps/newsletters-ui/src/app/components/Cell.tsx

```diff
@@ -1,13 +1,14 @@
 import { renderYesNo } from '../util';
 
 export type Cell<T> = { cell: { value: T } };
-
+export const MIGRATION_TIMESTAMP_VALUE = 946684800;
```

Comment:
We should probably define this value in the data-client library and import it from there in case we need it elsewhere